addons/ai/functions/fnc_mortarSupportFiredNear.sqf

```diff
@@ -0,0 +1,29 @@
+/*
+    Author:
+        Adam Bridgford
+
+    Description:
+        FiredNear EH fnc. Sets a cooldown so requests are made every 15secs.
+        Then does a check if mortar unit and RTO are alive.
+
+    Parameter(s):
+        0: Unit <OBJECT>
+
+    Return Value:
+        Nothing
+*/
+
+hint "firedNear fnc fired";
+if (!(_unit getVariable [QGVAR(unitCooldown), false])) then {
```

Comment:
```_unit``` does not refer to anything in the scope of this function. If you're passing a unit as a parameter to this function, access it here with ```params```